lib/core/elastic_connector_actions.rb

```diff
@@ -7,10 +7,12 @@
 # frozen_string_literal: true
 #
 require 'active_support/core_ext/hash'
-require 'utility/elasticsearch/index/mappings'
-require 'utility/elasticsearch/index/text_analysis_settings'
 require 'connectors/connector_status'
 require 'connectors/sync_status'
+require 'utility/elasticsearch/index/mappings'
+require 'utility/elasticsearch/index/text_analysis_settings'
+require 'utility/es_client'
+require 'utility/logger'
```

Comment:
Why not
```suggestion
require 'utility'
```

I also observed such cases before, even though there's no `require` for some files, it still doesn't throw any error.